### DISCUSSION TITLE
I'd like you to add a channel on Discord to report spam messages (such as Scam-Reports) and the support-ticket channel.

### DISCUSSION THREAD

**Discussion by dorikunz** [pkgxdev/pkgx#915] [Help]
It seems necessary to constantly post maliciously meaningless photos just for chat scores, and to have a related channel where community personnel can see and accuse the occasional scam channel in real time.

**Comment by michaelessiet** [comment]
Thank you for pointing this out. We'll make sure to look into this and filter out spam more thoroughly
